blimflix/client/index.html

```diff
@@ -26,13 +26,33 @@
     <![endif]-->
 
     <div class="container">
-      <div class="page-header">
-        <h2>Congratulations, you've scaffolded an app with Hestia!</h2>
-        <h4>I'd recommend starting by generating your first feature.</h4>
-        <div>Type this into your terminal: <kbd>ae feature your-first-feature</kbd></div>
-        <h4>After you make a feature, you'll probably need a controller to go along with it, and some HTML.</h4>
-        <div>Here's the command for that: <kbd>ae controller your-first-feature</kbd></div>
-      </div>
+        <div class="sidebar-wrapper col-xs-3">
+            <div class="row col-md-10 col-centered">
+                <div class="col-xs-3">
+                    <img class="img-responsive" src="./user.jpg" alt="" />
```

Comment:
Please, move images into ./client/images (a new folder).

Actually, let's remove it from the PR as @Migueld1324 told us he was going to do it on his branch.